napalm_logs/config/iosxr.yml

```diff
@@ -11,7 +11,7 @@ prefix:
   line: '{messageId}: {host}:{date} {time}: {processName}[{processId}]: %{error}'
 
 messages:
-  ROUTING-BGP-5-MAXPFX:
+  - error: ROUTING-BGP-5-MAXPFX
```

Comment:
I believe we should distinguish between the _napalm-logs error name_ and the _syslog error message_. With these I mean:

- the _napalm-logs error name_ will be a standard name for that error, that will be used to identify to error type (e.g.: BGP_PREFIX_THRESH_EXCEEDED points clearly). This name will be the same on all platforms
- the _syslog error message_ is the identification part from the raw syslog message (e.g.: ROUTING-BGP-5-MAXPFX).

Example:

```yaml
BGP_PREFIX_THRESH_EXCEEDED:
  error: ROUTING-BGP-5-MAXPFX
  values:
      peer: (\d+\.\d+\.\d+\.\d+)
      current: (\d+)
      limit: (\d+)
    line: 'No. of IPv4 Unicast prefixes received from {peer} has reached {current}, max {limit}'
    model: openconfig_bgp
    mapping:
      limit: bgp/neighbors/neighbor/{peer}/afi_safis/afi_safi/inet/ipv4_unicast/prefix_limit/state/max_prefixes
      current: bgp/neighbors/neighbor/{peer}/afi_safis/afi_safi/inet/state/prefixes/received
```
If the error field is not defined, it will default to the _napalm-logs error name_.
I think doing so it will scale better & it's more elegant.